### PR TITLE
Markdown Part 2

### LINKED ISSUE #74: Markdown Post Creation + Post Preview
Update the MakePost component to handle markdown text.

### LINKED ISSUE #75: Resize the textarea resize when writing
When doing markdown, it's hard to write content if we have a fixed height for the text-area. It should automatically adapt to the size of the content.

### PR DESCRIPTION
**Pull request by maharshmellow**
Fixes #74 
Fixes #75 

- write markdown in the MakePost component
- when the content is empty, the post and preview buttons are disabled
- clicking the preview button shows a modal with a preview of what your post will look like with the markdown rendered

The image is currently hardcoded. That will be fixed in another PR when the uploading of image is implemented.

### CODE REVIEW

File: src/styles/post/PostPreviewModal.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.post-preview-modal {
```
Comment (maharshmellow) [inline src/styles/post/PostPreviewModal.scss:1]: Will implement some styles later

File: src/tests/components/post/MakePost.test.jsx [hunk 2]
```diff
@@ -7,22 +7,4 @@ describe("Post box Component", () => {
     const component = shallow(<MakePost />);
     expect(component).toMatchSnapshot();
   });
-
-  it("should have a text area", () => {
-    const component = shallow(<MakePost />);
-    expect(component.find("textarea")).toHaveLength(1);
-  });
-
-  it("should show the modal after clicking on image icon", () => {
```
Comment (maharshmellow) [inline src/tests/components/post/MakePost.test.jsx:16]: This test wasn't actually doing anything - when changing the expected value to false it still passed 😃

**Review by zhaoyuxuan — APPROVED** [review]

File: src/styles/post/PostPreviewModal.scss (hunk 1 above)
Comment (ceciliaccwei) [reply src/styles/post/PostPreviewModal.scss:1]: I went through the code and ran the branch locally. Everything looks good! Is it possible to have some global styles for all modals? E.g. title font size?

**Review by ceciliaccwei — APPROVED** [review]

**Comment by maharshmellow** [comment]
@ceciliaccwei yess!! I have another issue up for making a shared sass file for all the common stylings between components so I’ll leave that for another PR.

**Pull request merged**